### DISCUSSION TITLE
What to do when `sudo add-apt-repository ppa:regolith-linux/release` said:

### DISCUSSION THREAD

**Discussion by floweringnights** [i3/i3#4598] [Q&A]
`Err:12 http://ppa.launchpad.net/regolith-linux/release/ubuntu impish Release`
  `404  Not Found [IP: 91.189.95.85 80]`
`Reading package lists... Done`                     
`E: The repository 'http://ppa.launchpad.net/regolith-linux/release/ubuntu impish Release' does not have a Release file.`
`N: Updating from such a repository can't be done securely, and is therefore disabled by default.`
`N: See apt-secure(8) manpage for repository creation and user configuration details.`?
I'm on Ubuntu 21.10 Impish Indri, and I'm trying to install i3-gaps. Should I be worried about this problem?

**Comment by Airblader** [comment]
I don't know how PPA works, but the same question for the i3 repository was asked recently [here](https://github.com/i3/i3/issues/4568), and the answer there was that the user misconfigured the entry. I assume this is the same thing, but with Regolith.

**Comment by kgilmer** [comment]
Regolith Desktop is now supported on Ubuntu Focal and Jammy and Debian Buster.  You can learn how to install the update here: regolith-desktop.com
